cibyl/sources/elastic/client.py

```diff
@@ -0,0 +1,72 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+import logging
+from elasticsearch import Elasticsearch
+from elasticsearch.exceptions import ConnectionError, ConnectionTimeout
+from sys import exit
+
+
+LOG = logging.getLogger(__name__)
+
+
+class ElasticSearchClient:
+
+    def __init__(self: object,
+                 host: str = "http://localhost",
+                 port: int = 9200) -> None:
+        self.host = f"{host}:{port}"
```

Comment:
maybe self.address or self.url?

Reply:
Yes, it's better to not repeat with the parameter. Done.